### ISSUE TITLE
Where is the linux tar/zip download?

### ISSUE THREAD

**Issue by delt0r** [pyfa-org/Pyfa#1628]
In the past i have just downloaded pyfa-VERSIONINFO-linux.zip. Yet this seems to no longer be available?

**Comment by blitzmann** [comment]
There is none. It was completely redundant in it's old state, as it was just the source zip with a single extra file included, and the source zip is automatically generated for download via GitHUb's release.

See https://github.com/pyfa-org/Pyfa/releases/tag/v2.0.0b1 for more info, under systen notes (and it also contains information you need to know if you're a linux user and just now upgrading to 2.x).

tl;dr: Just download the source zip :)
